src/cloudWatchLogs/commands/saveCurrentLogDataContent.ts

```diff
@@ -9,17 +9,18 @@ const localize = nls.loadMessageBundle()
 
 import * as fs from 'fs-extra'
 import { SystemUtilities } from '../../shared/systemUtilities'
-import { recordCloudwatchlogsDownloadStreamToFile, Result } from '../../shared/telemetry/telemetry'
+import { recordCloudwatchlogsDownload, CloudWatchResourceType, Result } from '../../shared/telemetry/telemetry'
 import { Window } from '../../shared/vscode/window'
-import { parseCloudWatchLogsUri } from '../cloudWatchLogsUtils'
-import { LogStreamRegistry } from '../registry/logStreamRegistry'
+import { isLogStreamUri, parseCloudWatchLogsUri } from '../cloudWatchLogsUtils'
+import { LogDataRegistry } from '../registry/logDataRegistry'
 
-export async function saveCurrentLogStreamContent(
+export async function saveCurrentLogDataContent(
     uri: vscode.Uri | undefined,
-    registry: LogStreamRegistry,
+    registry: LogDataRegistry,
     window = Window.vscode()
 ): Promise<void> {
     let result: Result = 'Succeeded'
+    let resourceType: CloudWatchResourceType = 'logStream' // Default to stream if it fails to find URI
```

Comment:
Kind of misleading on failures but this is a fault with how we do telemetry

Reply:
Yeah, I couldn't figure out another way to do this. Is there a way to change/improve the metric itself to avoid this?